worker/online_restore.go

```diff
@@ -246,7 +246,7 @@ func handleRestoreProposal(ctx context.Context, req *pb.RestoreRequest, pidx uin
 	}
 
 	if req.IncrementalFrom == 1 {
-		return errors.Errorf("Incremental restore must not include full backup")
+		req.IncrementalFrom = 0
```

Comment:
Could you illustrate how would this work if we need backups from 1 to say n. But not the full backup (which I guess would be from 0 to n).

Reply:
Full backup is number 1 and not 0. The PR should make sense to you now.